website/src/components/Providers/index.jsx

```diff
@@ -1,21 +1,23 @@
 "use client";
 import { MixpanelProvider } from "react-mixpanel-browser";
 import { HubspotProvider } from "next-hubspot";
-import { GoogleAnalytics } from "@next/third-parties/google";
```

Comment:
This wasn't even working.

Reply:
So we no longer use google analytics?

Reply:
We don't have a project for it, and we're already using Mixpanel for analytics. We already have GoogleAds and LinkedInInsidghts configured.

Reply:
As well as Hubspot analytics 🙃